advanced_source/cpp_extension.rst

```diff
@@ -1033,3 +1033,34 @@ mechanism as well as a motivation for using them. You can find the code examples
 displayed in this note `here
 <https://github.com/pytorch/extension-cpp>`_. If you have
 questions, please use `the forums <https://discuss.pytorch.org>`_.
+
+FAQ
+---
```

Comment:
@goldsborough it feels like the FAQ should go as a note into the API documentation, or into the FAQ in the API documentation.

Reply:
Ok